### PR TITLE
OHM-295 get visualizer API endpoint

### PR DESCRIPTION
**Pull request by rcrichton**
This PR includes the changes in #764 as it depends on these. Once that is merged in you will get a truer diff of the changes that are only for the endpoint. For now, you can just review the latest commit.

### CODE REVIEW

File: src/api/visualizers.coffee [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Visualizer = require('../model/visualizer').Visualizer
+authorisation = require './authorisation'
+utils = require '../utils'
+
+exports.getAllVisualizers = ->
+  # Must be admin
+  if not authorisation.inGroup 'admin', this.authenticated
+    utils.logAndSetResponse this, 403, 'User #{this.authenticated.email} is not an admin, API access to getAllVisualizers denied.', 'info'
```
Comment (hnnesv) [inline src/api/visualizers.coffee]: String should be using double quotes

File: src/api/visualizers.coffee [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    return
+
+  try
+    v = yield Visualizer.find().exec()
+    this.body = v
+  catch err
+    utils.logAndSetResponse this, 500, 'Could not fetch visualizers via the API: #{err}', 'error'
```
Comment (hnnesv) [inline src/api/visualizers.coffee]: ditto: string double quotes

**Comment by hnnesv** [comment]
LGTM. Just some minor comments. There's no need for further review after these and can be merged 👍

File: src/api/visualizers.coffee (hunk 1 above)
Comment (rcrichton) [inline src/api/visualizers.coffee]: Oops, this was some overzealous fins and replace...

**Pull request merged**